templates/data/index.html

```diff
@@ -0,0 +1,431 @@
+{% extends 'base_index.html' %}
+
+{% block title %}Data Fabric | Canonical{% endblock %}
+
+{% block meta_description %}Canonical Data Fabric Suite enables innovation at scale on any cloud. Advanced open source solutions for data on IaaS and Kubernetes with flexible, enterprise-grade support.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1HP9YdHF11yVQ9SLl0VXhLezDZREsr1GHIKNi9wT1Gv0/edit{% endblock meta_copydoc %}
+
+{% block body_class %}is-paper{% endblock body_class %}
+
+{% block content %}
+
+<section class="p-strip--whitesuru is-shallow u-no-padding--bottom">
+  <div class="p-section--deep">
+    <div class="row">
+      <div class="col-start-large-4 col-9">
+        <div class="p-section--shallow">
+          <h1>Enterprise Data Fabric solutions for rapid innovation at any scale</h1>
+        </div>
+        <div class="p-section--shallow">
+          <p>Data flows better with the right tools. Canonical’s open source enterprise Data Fabric suite delivers better ROI across clouds.<br />Engineered for the most critical workloads.</p>
+        </div>
+        <hr>
+				<div class="p-section">
+					<p>
+						<a class="p-button--positive js-invoke-modal" href="/data/contact-us#get-in-touch" aria-controls="data-fabric-modal">Get in touch</a>
+					</p>
+				</div>
+      </div>
```

Comment:
This tabbing looks off

Reply:
I fixed my vscode tabbing settings, shouldn't be a problem anymore